parse_lib.py

```diff
@@ -95,8 +94,9 @@ def resolve_relative_resource_urls(html_string):
     imgs = html.find_all("img", src=True)
     equations = html.find_all("object", data=True)
     list(map(update_anchor_href, anchors))
```

Comment:
This is a subtle point, so @yujanshrestha and @andrewemark what do you think about my argument against using `map` like this?